addons/website/tests/test_views.py

```diff
@@ -17,6 +17,16 @@ def attrs(**kwargs):
     return {'data-oe-%s' % key: str(value) for key, value in kwargs.items()}
 
 
+def _create_imd(self, view):
```

Comment:
Yuck!
Why not create an intermediate class with this function?

Reply:
I thought it would be overkill just for that helper method, but I did the change, it is cleaner anyway :+1: